src/lib/clients/keeper.ts

```diff
@@ -1,43 +1,39 @@
-import { CommandClient, Handler } from "../../types/generic";
+import { CommandClient, Command } from '../../types/generic';
 import { KeeperClientOptions, KeeperSlackMiddleware } from '../../types/clients/keeper'
-import SlackClient from "./slack";
-import KeeperCommandHandlerMap from "../handlers/keeper";
-import { Logger } from "../../types/logger";
+import KeeperCommandHandlerMap from '../handlers/keeper';
+import SlackClient from './slack';
+import PhonyClient from './phony';
+import { Logger } from '../../types/logger';
 
 class KeeperClient implements CommandClient<KeeperClient> {
     private logger: Logger;
-    private slackClient: SlackClient;
-    private slackCommandHandlers: { [command: string]: Handler<KeeperSlackMiddleware> }
+    private slackClient: CommandClient<SlackClient | PhonyClient>;
+    private slackCommandHandlers: Command<KeeperSlackMiddleware, this>[];
 
     constructor(options: KeeperClientOptions) {
         this.logger = options.logger;
         this.slackClient = options.slackClient;
         this.slackCommandHandlers = KeeperCommandHandlerMap;
     }
 
-    start = (): Promise<void> => {
-        return this.slackClient
-            .start()
-            .then(this.addDefaultCommandHandlers)
-            .then(() => {
-                this.logger.log(`Started ${this.constructor.name}#start():SUCCESS`)
-            })
+    start = async (): Promise<void> => {
+        await this.slackClient.start();
+        this.addDefaultCommandHandlers();
+        this.logger.log(`Started ${this.constructor.name}#start():SUCCESS`);
     }
 
-    stop = (): Promise<void> => {
-        return this.slackClient.stop();
+    stop = async (): Promise<void> => {
+        await this.slackClient.stop();
```

Comment:
can drop the `await`  here

Reply:
The `stop` method returns a promise, is there any reason we wouldn't want to `await` that?

Reply:
forgot, thanks :)